androidGeoBot/app/src/main/res/layout/activity_manual.xml

```diff
@@ -194,23 +117,24 @@
                 android:layout_height="50dp"
                 app:layout_constraintBottom_toBottomOf="parent"
                 android:layout_marginBottom="25dp"
+                android:fontFamily="@font/bree_serif"
                 android:gravity="center">
-                <EditText
+
+                <TextView
                     android:id="@+id/editTextTextPersonName2"
```

Comment:
What does this id mean? It would probably be better if we gave this an id of "timer" or something similar.

Reply:
All fixed now! Thanks for letting me know ☺